orga/app/styles/components/manage-authentication-method-modal.scss

```
@@ -108,7 +108,11 @@
   &__subTitle {
     display: flex;
     justify-content: space-between;
-    margin-bottom: 16px;
+    margin-bottom: $pix-spacing-s;
+
+    &--mediacentre {
+      margin-bottom: 0px;
```

```suggestion
      margin-bottom: 0;
```

cf. https://www.quora.com/Which-is-better-0px-or-0

---

j'en ai profité pour enlever les autres présents dans le fichier dans un commit de refactor